Session/Conversations/Input View/InputView.swift

```diff
@@ -141,6 +141,10 @@ final class InputView: UIView, InputViewButtonDelegate, InputTextViewDelegate, M
     }()
 
     private lazy var additionalContentContainer = UIView()
+    
+    public var isKeyboardVisible: Bool {
```

Comment:
Calling this `isKeyboardVisible` seems a little misleading - could it just be `isInputFirstResponder`? (assuming `isFirstResponder` is reserved and would cause headaches if we used it)